src/allmydata/version_checks.py

```diff
@@ -23,11 +23,21 @@
     verlib,
 )
 
-_INSTALL_REQUIRES = list(
-    str(req)
-    for req
-    in pkg_resources.get_distribution(__appname__).requires()
-)
+if getattr(sys, 'frozen', None):
+    # "Frozen" python interpreters (i.e., standalone executables
+    # generated by PyInstaller and other, similar utilities) run
+    # independently of a traditional setuptools-based packaging
+    # environment, and so pkg_resources.get_distribution() cannot be
+    # used in such cases to gather a list of requirements at runtime
+    # (and because a frozen application is one that has already been
+    # "installed", an empty list suffices here).
+    _INSTALL_REQUIRES = []
```

Comment:
To get this line marked as covered, I suppose you would have to collect coverage from the `dist/Tahoe-LAFS/tahoe --version` call which - I imagine - doesn't work?

Reply:
No, I don't think that would work. Under a "frozen" interpreter/environment, the original python source module files are no longer present (and so, presumably, coverage tools would have no way of referencing which lines are being hit).